### PR TITLE
Tensorflow 2 changes

### PR DESCRIPTION
**Pull request by franzigeiger**
To make candidate models trensorflow V2 compatible I've run their provided script and this is the result. 
It won't work yet though because some dependencies have the same issues.

### CODE REVIEW

**Comment by franzigeiger** [comment]
The tests fail because of brain-score. My fix PR has to be merged first

File: candidate_models/base_models/__init__.py [hunk 1]
```diff
@@ -448,9 +444,9 @@ def __init__(self):
             self[identifier] = LazyLoad(function)
 
     def __getitem__(self, basemodel_identifier):
-        if basemodel_identifier in self._accessed_base_models:
-            raise ValueError(f"can retrieve each base model only once per session due to possible hook clashes - "
-                             f"{basemodel_identifier} has already been retrieved")
+        # if basemodel_identifier in self._accessed_base_models:
+        #     raise ValueError(f"can retrieve each base model only once per session due to possible hook clashes - "
+        #                      f"{basemodel_identifier} has already been retrieved")
```
Comment (mschrimpf) [inline candidate_models/base_models/__init__.py]: dead code

File: candidate_models/model_commitments/cornets.py [hunk 2]
```diff
@@ -356,20 +355,20 @@ def __init__(self):
...
-                        raise ValueError(f"{identifier}'s brain-model {basemodel_identifier} has already been accessed "
-                                         f"in this session. To avoid clashes in the hooks, "
-                                         f"please run {identifier} in a separate session.")
-                    self._accessed_brain_models.append(basemodel_identifier)
-                    # upon accessing the `activations_model`, the Hook will automatically
-                    # attach to the `brain_model.activations_model`.
-                    return brain_model
-
-                self[identifier] = LazyLoad(load)
+            # for identifier, activations_model in Hooks().iterate_hooks(basemodel_identifier, activations_model):
```
Comment (mschrimpf) [inline candidate_models/model_commitments/cornets.py]: dead code

**Comment by franzigeiger** [comment]
I have resolved the issues, it is only tensorflow api changes now

File: setup.py [hunk 3]
```diff
@@ -15,7 +15,11 @@
     "networkx==1.11",
     "tqdm",
     "gitpython",
-
+    "torch",
+    "keras",
+    "pandas==0.25.3",
+    "tensorflow==1.15",
```
Comment (mschrimpf) [inline setup.py]: Is there another solution than requiring users to `pip install` all 3 (pytorch, keras, tensorflow)?

**Pull request merged**